docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_DeepSpeech_From_Tensorflow.md

```diff
@@ -2,66 +2,73 @@
 ```
-wget -O - https://github.com/mozilla/DeepSpeech/releases/download/v0.3.0/deepspeech-0.3.0-models.tar.gz | tar xvfz -
+wget -O - https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz | tar xvfz -
+wget -O - https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz | tar xvfz -
 ```
 * For Windows* systems:
-  1. Download the archive from the DeepSpeech project repository: [https://github.com/mozilla/DeepSpeech/releases/download/v0.3.0/deepspeech-0.3.0-models.tar.gz](https://github.com/mozilla/DeepSpeech/releases/download/v0.3.0/deepspeech-0.3.0-models.tar.gz).
-  2. Unpack it with a file archiver application.
+  1. Download the archive with the model: [https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz](https://github.com/mozilla/DeepSpeech/archive/v0.8.2.tar.gz).
+  2. Download the TF metagraph with pretrained weights: [https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz](https://github.com/mozilla/DeepSpeech/releases/download/v0.8.2/deepspeech-0.8.2-checkpoint.tar.gz).
+  3. Unpack it with a file archiver application.
+
+## Freeze the model into a *.pb file
 
-After you unpack the archive with the pre-trained model, you will have the new `models` directory with the
-following files:
+After you unpack the archives below, you have to freeze the model. Please note that this requires 
+TensorFlow* version 1 which is not available under python 3.8, so you will need python 3.7 or lesser.
+Before freezing deploy a virtual environment and install required packages:
 ```
-alphabet.txt  
-lm.binary
-output_graph.pb  
-output_graph.pbmm  
-output_graph.rounded.pb  
-output_graph.rounded.pbmm  
-trie
+virtualenv --python=python3.7 venv-deep-speech
+source venv-deep-speech/bin/activate
+cd DeepSpeech-0.8.2
+pip3 install -e .
 ```
+Freeze the model with the following command:
+```
+python3 DeepSpeech.py --checkpoint_dir ../deepspeech-0.8.2-checkpoint --export_dir ../
+```
+After that you will get the pretrained frozen model file `output_graph.pb` in the directory `deepspeech` created at 
+the beginning. The model contains the preprocessing and main parts. The first preprocessing part performs conversion of input 
+spectrogram into a form that's useful for speech recognition (mel). This part of the model is not convertible into 
+IR because it contains 2 unsupported operations `AudioSpectrogram` and `Mfcc`. 
 
-Pre-trained frozen model file is `output_graph.pb`.
-
-![DeepSpeech model view](../../../img/DeepSpeech.png)
-
-As you can see, the frozen model still has two variables: `previous_state_c` and
-`previous_state_h`. It means that the model keeps training those variables at each inference.
-
-At the first inference of this graph, the variables are initialized by zero tensors. After executing the `lstm_fused_cell` nodes, cell state and hidden state, which are the results of the `BlockLSTM` execution, are assigned to these two variables.
```

Comment:
I am not sure about why is this paragraph is skipped. How rnn states are initialized from one inference request to another?

Reply:
cell and hidden state variables must be initialized in the user script https://github.com/openvinotoolkit/openvino/pull/6011/files#diff-80d5d3287334e6c55b8e6245824b388f995a62a81fc566ccd08d744820996934R61-R62

I reverted part of this paragraph with picture and clarification about  `previous_state_c` and `previous_state_h`